x-pack/plugins/observability_solution/dataset_quality/public/components/common/descriptive_switch.tsx

```diff
@@ -13,17 +13,26 @@ interface DescriptiveSwitchProps {
   checked: boolean;
   tooltipText: string;
   onToggle: () => void;
+  dataTestSubject: string;
```

Comment:
Sorry for nitpicking. Just `testSubject` could be a bit better name, `data` part is kind of a implementation detail.